staging/src/k8s.io/apiserver/pkg/endpoints/handlers/update.go

```diff
@@ -72,6 +73,11 @@ func UpdateResource(r rest.Updater, scope RequestScope, admit admission.Interfac
 			scope.err(err, w, req)
 			return
 		}
+		if errs := validation.ValidateUpdateOptions(options); len(errs) > 0 {
+			err := errors.NewInvalid(schema.GroupKind{Group: metav1.GroupName, Kind: "UpdateOptions"}, "", errs)
+			scope.err(err, w, req)
```

Comment:
what http status code do these errors produce?

Reply:
I suspect it's `http.StatusUnprocessableEntity`

Reply:
yes, 422. that matches what we do for invalid objects, so that's good. should probably update the field doc for DryRun to be more generic ("...will result in an error response...") rather than say it will be a BadRequest error code.

Reply:
Good catch